ui/components/Navigator.js

```diff
@@ -834,13 +839,13 @@ class Navigator extends React.Component {
       logoIcon = <img src={image} className={classes.icon} style={{ filter: filter }} />;
     }
     return logoIcon;
-  }
+  };
 
   /**
    * Changes the route to "/"
    */
-  handleTitleClick = () => {
-    this.props.router.push('/');
+  const handleTitleClick = () => {
+    props.router.push('/');
```

Comment:
Why we are using router from props, might be because of class based component but as we moved to functional based let's not use router from prop instead from hook useRouter()

Reply:
done

Reply:
Thankyou